src/Hyperion/Extensions/TypeEx.cs

```diff
@@ -132,19 +144,82 @@ private static Type GetTypeFromManifestName(Stream stream, DeserializerSession s
             });
         }
 
+        public static bool disallowUnsafeTypes = true;
```

Comment:
So, you can't -really- get to this since TypeEx is internal. This should probably be either moved or left out (because really, -why- would you want to do any of these? it's just a bad idea.)